packages/firestore/lite/src/api/components.ts

```diff
@@ -81,6 +84,6 @@ export async function removeComponents(firestore: Firestore): Promise<void> {
   if (datastorePromise) {
     logDebug(LOG_TAG, 'Removing Datastore');
     datastoreInstances.delete(firestore);
-    return (await datastorePromise).termiate();
+    return (await datastorePromise).terminate();
```

Comment:
s/dataPromise/datastoreInstance?

You don't need the `await` anymore since, it's no longer a promise.

Reply:
Done